docs/release-notes/auto-populate-posix.rst

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+**New Features**
+
+-  Master Configuration: Master Configuration accepts POSIX claims `agent_uid_attribute_name`,
+   `agent_gid_attribute_name`, `agent_user_name_attribute_name`, or
+   `agent_group_name_attribute_name`. Please visit :ref:`oidc master configuration
+   <_master-config-oidc>` or :ref:`saml master configuration <_master-config-saml>` for details. If
+   any of the fields are configured, they will be used and synced to the database.
```

Comment:
I don't understand why it's accepting `_master-config-saml` as a reference link but not `_master-config-oidc`.

Reply:
:orphan:

**New Features**

-  Master Configuration: Add support for POSIX claims in the master configuration. It now accepts 
   ``agent_uid_attribute_name``, ``agent_gid_attribute_name``, ``agent_user_name_attribute_name``, 
   or ``agent_group_name_attribute_name``. Refer to the :ref:`OIDC master configuration <master-config-oidc>` 
   or :ref:`SAML master configuration <master-config-saml>` for details. If any of these fields 
   are configured, they will be used and synced to the database.

Reply:
`:orphan:

**New Features**

-  Master Configuration: Add support for POSIX claims in the master configuration. It now accepts 
   ``agent_uid_attribute_name``, ``agent_gid_attribute_name``, ``agent_user_name_attribute_name``, 
   or ``agent_group_name_attribute_name``. Refer to the :ref:`OIDC master configuration <master-config-oidc>` 
   or :ref:`SAML master configuration <master-config-saml>` for details. If any of these fields 
   are configured, they will be used and synced to the database.
`

Reply:
@kkunapuli i think maybe you added the leading underscore and its that part that is not needed. maybe something with the angle brackets. https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
That was it - thank you so much @tara-det-ai!